tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py27,py35,pep8
```

Comment:
The envlist is just the set of environments that are run when you just do `tox` without a `TOXENV` env var and without a `-e` argument. Since we run tox with `TOXENV` to target specific environments (which allows us to parallelize the tests somewhat) travis effectively ignores this.